### ISSUE TITLE
How to train CAM?

### ISSUE THREAD

**Issue by qi0416** [metalbubble/CAM#52]
if i want to train a cam which is used medical image（3 class），which loss is my need？

**Comment by qi0416** [comment]
loss = torch.nn.MultiLabelSoftMarginLoss()?
